src/App.tsx

```diff
@@ -17,18 +75,24 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter filter={filter} onFilter={setFilter} />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading ? (
+                <Loader />
+              ) : (
+                <>
+                  <TodoList todos={filtredTodos} />
+                  {!filtredTodos.length && 'No todos'}
```

Comment:
```suggestion
                 {filtredTodos.length > 0 ? (
                    <TodoList todos={filtredTodos} />
                  ) : (
                    'No todos'
                  )}
```
Looks like if you have no todos you don't need to render `TodoList` at all